isis/tests/CnetWinnowTests.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+#include <QTemporaryDir>
+
+#include "cnetwinnow.h"
+
+#include "PixelType.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "PvlKeyword.h"
+#include "TestUtilities.h"
+#include "FileName.h"
+#include "SerialNumber.h"
+
+#include "Fixtures.h"
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/cnetwinnow.xml").expanded();
+
+TEST_F(ThreeImageNetwork, CnetwinnowFunctionalDefault) {
```

Comment:
Since I updated the docs to mention naming format last night, it saves me time renaming things later if you change this to FunctionalTestCnetwinnowDefault unless anyone has an issue with this format.